packages/tenant-readmodel-writer/src/tenantConsumerService.ts

```diff
@@ -65,11 +65,11 @@ export async function handleMessage(
       await tenants.updateOne(
```

Comment:
this event is useful only in akka to allow data retrieval between different shards.
Moreover, is used only when deleting a tenant.

We can consider it a no-op and remove the update

Reply:
Updated